mathesar/models/base.py

```diff
@@ -656,6 +661,11 @@ def insert_records_to_existing_table(self, existing_table, data_files, mappings=
             raise e
         return table
 
+    def get_mappings(self, existing_table):
```

Comment:
Please give this a more descriptive name. There are many meanings for `mappings`, `mapper`, etc.